test/wdio/reflect-nan-attribute-with-child/cmp.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('reflect-nan-attribute-with-child', () => {
+  beforeEach(() => {
+    render({
+      template: () => <parent-reflect-nan-attribute></parent-reflect-nan-attribute>,
+    });
+  });
+
+  it('renders the parent and child the correct number of times', async () => {
+    await $('parent-reflect-nan-attribute').waitForExist();
+
+    await expect($('parent-reflect-nan-attribute').$('>>>div:first-of-type').$('div:first-of-type')).toHaveText(
```

Comment:
Doing something like:
```tsx
    await expect($('parent-reflect-nan-attribute')).toHaveText("...");
```
Gave me the text of both the parent and child component here, which was a bit unexpected, as I thought I'd have to pierce the shadow dom to get the text of the child. Maybe that's not the case?

Anyway, that's why these selectors are a bit longer - happy to change them if there's a better way!